service/create/pki.go

```diff
@@ -0,0 +1,119 @@
+package create
+
+import (
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+
+	"github.com/giantswarm/certctl/service/pki"
+	"github.com/giantswarm/certctl/service/token"
+)
+
+const (
+	// TODO Choose a sensible value and move to ConfigMap for V1
+	CertificateAuthorityTTL = "1440h"
```

Comment:
Yes it makes sense that the CA shouldn't expire unless there is a breach. I'll move this to a config map.